test/Microsoft.ML.AutoML.Tests/Microsoft.ML.AutoML.Tests.csproj

```diff
@@ -21,6 +21,7 @@
     </None>
   </ItemGroup>
 
+  <!-- TODO: once we fix the 4 intel MKL methods, SymSgdNative will need to go back in. -->
```

Comment:
I still see these TODOs in a few places. 

* test/Microsoft.ML.IntegrationTests/Microsoft.ML.IntegrationTests.csproj
* test/Microsoft.ML.NightlyBuild.Tests/Microsoft.ML.NightlyBuild.Tests.csproj
* test/Microsoft.ML.Predictor.Tests/Microsoft.ML.Predictor.Tests.csproj

Should they all be removed?

Reply:
Yes, and they should be now.